components/cards/cards.css

```diff
@@ -0,0 +1,146 @@
+body {
+    background-color: var(--DEE2E6);
+}
+
+
+/* useless CSS prior to this */
+.card-shadow {
+    box-shadow: 0 4px 8px 0 var(--PURE-BLACK), 0 6px 20px 0 var(--PURE-BLACK);
+}
+
+.card-element-horizontal {
+    width: 35rem;
+    background-color: var(--PURE-WHITE);
+    border-radius: 4px;
+}
+
+.card-element-horizontal>br {
+    color: var(--DEE2E6);
+}
+
+/* card element main content */
+.card-element__main {
+    display: flex;
+
+}
+
+.card__image-container {
+    width: 11rem;
+    height: 11rem;
+
+}
+
+.card__image-container>img {
+    border-radius: px4px 4px 0 0;
```

Comment:
I think there's a typo here.

Reply:
Oh! yeah have not seen it thanks man
Thanks for reviewing it too